SCSH_3C_iOS/Networking/Moya/MoyaApiInterface/Auth/TokenApiInterface.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TokenApiInterface.swift
+//  Networking
+//
+//  Created by 辜敬閎 on 2023/10/2.
+//
+
+import Foundation
+import Moya
+
+public struct TokenApiInterface: MoyaApiInterfaceType {
```

Comment:
So.... an API interface only can handle only one specific method, header and JSON encoder/decoder? 

If there is another `DELETE api/v1/token` api and we need another one API interface?

Reply:
> So.... an API interface only can handle only one specific method, header and JSON encoder/decoder?

For the JSON encoder/decoder section, Yes. 
We rarely need more than one code format in this context, especially since the server is not an external system.

> If there is another DELETE api/v1/token api then we need to create another one API interface?

It depends, For a quick solution, I can handle DELETE request by simply changing the property `method`  to `.delete`, However, if this becomes a frequent requirement, I'd consider refactoring it with a nested structure.